geoportal/c2cgeoportal_geoportal/scaffolds/create/geoportal/Dockerfile_tmpl

```diff
@@ -32,13 +32,11 @@ RUN true
 COPY --from=builder /usr/lib/node_modules/ngeo/dist/* /etc/static-ngeo/
 COPY --from=builder /etc/static-ngeo/* /etc/static-ngeo/
 COPY --from=builder /app/alembic.ini /app/alembic.yaml ./
-RUN chmod go+w /etc/static-ngeo/ \
-    /app/{{package}}_geoportal/locale/ \
-    /app/{{package}}_geoportal/locale/*/LC_MESSAGES/{{package}}_geoportal-client.po
+RUN chmod go+w /etc/static-ngeo/
 
 RUN python3 -m pip install --disable-pip-version-check --no-cache-dir --editable=/app/ && \
     python3 -m compileall -q /usr/local/lib/python3.8 \
-        -x '/usr/local/lib/python3.8/dist-packages/(pydevd|ptvsd|pipenv)/' && \
+        -x '/usr/local/lib/python3.8/dist-packages/(pydevd|ptvsd|pipenv|.*pydev.*)/' && \
```

Comment:
For curiosity why do need to do this change?

Reply:
Did not seen that we already have `pydevd` in the regex (can be removed as .*pydev.* also match).
Pydevd comes with some other modules : 

_pydev_bundle
_pydev_imps
_pydev_runfiles
_pydevd_bundle
_pydevd_frame_eval

And `compileall` fails on these folders.

Reply:
We can keep that but it is curious why it's failing now...